### PR TITLE
NOISSUE: Improve prev pulse handling

### PR DESCRIPTION
**Pull request by cyraxred**
* Added method to properly get prev _operational_ pulse
* Corrected test server to avoid "hanging/first" pulse issue

### CODE REVIEW

File: ledger-core/virtual/integration/utils/server.go [hunk 1]
```diff
@@ -152,6 +152,7 @@ func newServerExt(ctx context.Context, t Tester, errorFilterFn logcommon.ErrorFi
 	s.pulseManager = PulseManager
 	s.pulseStorage = Pulses
 	s.pulseGenerator = testutils.NewPulseGenerator(10)
+	s.incrementPulse(ctx) // for sake of simplicity make sure that there is no "hanging" first pulse
```
Comment (bigbes) [inline ledger-core/virtual/integration/utils/server.go:155]: Can we move that part to init?

Comment (sexyhamster) [reply ledger-core/virtual/integration/utils/server.go:155]: this one could be removed, because in Init we already do IncerementPulse

Comment (cyraxred) [reply ledger-core/virtual/integration/utils/server.go:155]: we need 2 increments (first will have no valid prev)

Comment (cyraxred) [reply ledger-core/virtual/integration/utils/server.go]: moved to init

**Pull request merged**